### PR TITLE
[WIP] Refactor seeding lib

### PR DESCRIPTION
**Pull request by DerekStride**
How does this refactor look? It'll make building these seeders a bit easier.

### CODE REVIEW

File: lib/tasks/seed.rake [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   desc "Create users with data from ffaker."
   task users: :environment do
-    faker = Faker.new(count)
-    faker.create_users
+    faker = Faker::Users.new(count)
+    faker.create!
   end
 end
 
 def count
```
Comment (justinkrol) [inline lib/tasks/seed.rake]: We should pass in a symbol for the task, so it returns the right environment variable, or just name this `count_users` for now.
E.g.
```
def count(task_name)
  ENV.fetch("NUM_#{task_name.upcase}", default(task_name)
end

def default(task_name)
  case task_name
  when :users
    50
  ...
end
```
Or we could just stick with a single case statement
```
def count(task_name)
  case task_name
  when :users
    ENV.fetch("NUM_USERS", 50)
  ...
end
```

File: lib/tasks/seed.rake [hunk 2]
```diff
@@ -1,13 +1,13 @@
-require 'faker'
+require 'faker/user'
```
Comment (justinkrol) [inline lib/tasks/seed.rake:1]: Probably in separate change, but maybe we can add the `require_all` gem and `require_all 'faker'`?

Comment (DerekStride) [reply lib/tasks/seed.rake:1]: Does shopify have custom autoloading/eagerloading code for the lib directory?

Comment (justinkrol) [reply lib/tasks/seed.rake:1]: Let's take a look on Tuesday.

**Comment by justinkrol** [comment]
This is blocking for adding the rest of the seeders. Gonna merge, please review and we'll just fix forward anything that needs to change.

**Pull request merged**